src/bramble/client/main.js

```diff
@@ -920,6 +925,10 @@ define([
         this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_DISABLE_WORD_WRAP"}, callback);
     };
 
+    BrambleProxy.prototype.configureAutoCloseTags = function(options, callback) {
+        this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_CONFIGURE_AUTO_CLOSE_TAGS", args: [ options ]}, callback);
```

Comment:
There are two other bindings though that use the same syntax. I think I'd have to modify how those work as well. Same as in RemoteCommanderHandler.js as well.